metagpt/configs/llm_config.py

```diff
@@ -94,6 +94,10 @@ class LLMConfig(YamlModel):
     # For Network
     proxy: Optional[str] = None
 
+    # For baidu qianfan
+    system: bool = False       # 模型人设
+    disable_search: bool = True   # 是否关闭实时搜索
```

Comment:
The modification must include the actual parameter changes of the underlying API call, otherwise it will not take effect.